doc/Curvilinear_grid.ipynb

```diff
@@ -81,7 +81,7 @@
     }
    ],
    "source": [
-    "ds = xr.tutorial.load_dataset('rasm')\n",
+    "ds = xr.tutorial.open_dataset('rasm').load()\n",
```

Comment:
Let's add a note for older versions and do not call `load`
```
ds = xr.tutorial.open_dataset('rasm') # use xr.tutorial.load_dataset() for xarray<v0.11.0
```

Same for https://github.com/JiaweiZhuang/xESMF/blob/c123a5bd574bf538844ea89bcb3528de3441a73e/doc/Rectilinear_grid.ipynb#L77